docs/cluster-agent/README.md

```diff
@@ -0,0 +1,52 @@
+# Datadog Cluster Agent - DCA | User documentation
+
+The DCA is a **beta** feature, if you are facing any issues please reach out to our [support team](http://docs.datadoghq.com/help)
+
+## Introduction
+
+In the context of monitoring Orchestrators, solely relying on insight from the node is not enough.
+Kubernetes, DCOS, Swarm etc are working at the node level but also at the cluster level.
+The Datadog agent has the capability to fully monitor a node at the system and the application level, it also gives good insights of the cluster's health.
+Nevertheless, in order to separate concerns it is important to keep the node agent to the context of the node and have a cluster agent take care of the higher level.
+
+## Monitoring a cluster
+
+From the cluster level, users should care about the events as well as the cluster level metadata.
+For instance, the services serving a certain pod living on a node.
+
+## Before the DCA
+
+Node agents would run a leader election process among each other, the leader would query the API Server on a regular basis to collect the kubernetes events.
+Each agent would query the API server on their own to get the services serving the pods on their node and map them to tag the kubernetes metrics with the appropriate pod name and service associated.
+The limits of such an approach are:
+- Non linear increase of the load on the API Server and ETCD
+- Error prone process that can lead to a duplicate collection of events. 
+
+## Enters the DCA
+
+The goal of the DCA is to enhance the experience of monitoring Kubernetes:
+
+* It acts as a proxy between the API server and the node agent in order to separate concerns.
+* It provides cluster level metadata that can only be found in the API server to the node agents for them to enrich the metadata of the locally collected metrics.
+* It enables a cluster level collection of data such as the monitoring of services or SPOF and events that could otherwise than via a mix of [Leader Election] and [Autodiscovery] could not be monitored.
+
```

Comment:
I think that's the "value" point of this docs; Ideally I'd love to have a diagram here showing the communications between the DCA and the ApiServer & the node agents and more importantly showing _what_ is communicated.